### PR TITLE
Text updates.

### PR DESCRIPTION
**Pull request by BethMyre**

### CODE REVIEW

File: parser.go [hunk 1]
```diff
@@ -191,66 +191,39 @@ func haveAllItems() bool {
...
+	slide :: Travel quickly.
+	yank :: See take.
+	shrink :: Make a big thing a smaller thing.
+	cut :: Cut an item.
+	savegame :: Saves the state of the game to a file.
+	loadgame :: Confirms that this really is desired, then loads
+		the game state from a file.
+	exit :: Save game and then exit.
+	quit :: See exit.
+	help :: Print this message.`)
```
Comment (ewk) [inline parser.go:249]: This is less readable than what we had.

File: parser.go [hunk 2]
```diff
@@ -318,7 +318,6 @@ func eatItem(item string) {
 	if val, ok := inventory[item]; ok {
 		if val.IsEdible {
 			fmt.Println("That was delicious! Your strength has been restored.")
-			delete(inventory, item)
```
Comment (ewk) [inline parser.go:320]: I'm not on board with this. If you eat an edible item, it's gone forever. (Or until your parents go to the store and get more.)

Comment (mccarrom) [reply parser.go:320]: But you will lose automatically if you have eaten the corn flakes, which you need to fix the shrink ray and are also the only edible thing left, and the player has no way of knowing this.

File: parser.go (hunk 1 above)
Comment (BethMyre) [reply parser.go:249]: I added the blank lines back.

File: parser.go (hunk 2 above)
Comment (BethMyre) [reply parser.go:320]: I changed the message that's printed.

**Comment by ewk** [comment]
Since we're down to the only edible item I guess it doesn't matter if we keep the ability to also delete it.

**Pull request merged**